src/core/service.c

```diff
@@ -1263,10 +1262,6 @@ static int service_spawn(
         if (r < 0)
                 return r;
 
-        r = unit_full_printf_strv(UNIT(s), c->argv, &argv);
-        if (r < 0)
```

Comment:
Hm, we called `unit_realize_cgroup`, `unit_full_printf_strv`. So, `%c` worked.
This commit breaks `%c`:
```
bash-4.3# systemctl cat a --no-pager
# /etc/systemd/system/a.service
[Service]
ExecStart=/bin/sh -x -c 'echo %c'

bash-4.3# systemctl start a

bash-4.3# journalctl -u a --no-pager -o cat
...
a.service: About to execute: /bin/sh -x -c 'echo /a.service'
...
```
Should be `/system.slice/a.service`

BTW: I didn't check other specifiers.
Also, I think we should add some regression tests